js/base/Exchange.js

```diff
@@ -365,15 +365,20 @@ module.exports = class Exchange {
         return this.fetch2 (path, type, method, params, headers, body)
     }
 
-    parseJson (responseBody, url, method = 'GET') {
+    parseJson (response, responseBody, url, method) {
```

Comment:
This called after request has been made, method is always defined at this stage.